server/src/test/java/io/crate/integrationtests/SysSnapshotsTest.java

```diff
@@ -104,13 +105,14 @@ public void testQueryAllColumns() throws Exception {
         assertThat((List<Object>) firstRow[0]).containsExactly(getFqn("tbl"));
         assertThat((List<Object>) firstRow[1]).isEmpty();
         assertThat((Long) firstRow[2]).isLessThanOrEqualTo(finishedTime);
-        assertThat(firstRow[3]).isEqualTo("s1");
-        assertThat(firstRow[4]).isEqualTo("r1");
-        assertThat((Long) firstRow[5]).isGreaterThanOrEqualTo(createdTime);
-        assertThat(firstRow[6]).isEqualTo(SnapshotState.SUCCESS.name());
-        assertThat((List<Object>) firstRow[7]).isEmpty();
-        assertThat((List<Object>) firstRow[8]).containsExactly(getFqn("tbl"));
-        assertThat(firstRow[9]).isEqualTo(Version.CURRENT.toString());
+        // firstRow[3] is UUID, not selecting it as it's random.
+        assertThat(firstRow[4]).isEqualTo("s1");
+        assertThat(firstRow[5]).isEqualTo("r1");
+        assertThat((Long) firstRow[6]).isGreaterThanOrEqualTo(createdTime);
+        assertThat(firstRow[7]).isEqualTo(SnapshotState.SUCCESS.name());
+        assertThat((List<Object>) firstRow[8]).isEmpty();
+        assertThat((List<Object>) firstRow[9]).containsExactly(getFqn("tbl"));
+        assertThat(firstRow[10]).isEqualTo(Version.CURRENT.toString());
```

Comment:
Positions of other columns is different (I think we use alphabetical order currently).
Does it make it a breaking change?

Should I instead assign ordinals according to current order and add a new column with the max ordinal?
Will of course change ordering to rely on ordinal if I assign ordinals

Reply:
I think we usually don't treat an added column in a system table as a breaking change. But we should add a changes entry about the added column. I think we can leave the ordering as is.

Reply:
On change entry: I was going to hold on this change and later add this to the 5.9 because new column is a feature and we cannot merge now? See https://github.com/crate/crate/pull/16248#issuecomment-2199503274

Reply:
>I think we usually don't treat an added column in a system table as a breaking change.

Yep, was just concerned that `name` and `repository`  columns appear in different positions in  `select * from sys.snapshots` output (as shown in https://github.com/crate/crate/pull/16248/files#diff-ad636938a6a7441583ed16ac84f2f79895fd5fe7ba75f9ef205beca70d8fd255R83)

An app accessing those columns from `ResultSet` by index can be broken (but that's probably very narrow use case)